gateway_to_backend/README.md

```diff
@@ -518,6 +545,7 @@ definition)
 
     gw-request/otap_set_target_scratchpad/<gw-id>/<sink-id>
 
+    gw-request/set_configuration_data_item/<gw-id>/<sink-id>
```

Comment:
Now thinking of it, I do think that we should probably have similar get API for this. Then it would be more analogical with other setter type of API methods (you can read all of those with corresponding get method). While that is to some extent available at the moment as well, there is just extra overhead if we would need to query whole sink configs if some backend component is just interested on its own appconfig type of application configuration etc.